src/cli/index.ts

```diff
@@ -35,6 +35,10 @@ program
         model.consume(object);
       }
 
+      while (model.hasPendingTransactions) {
+        model.consumePendingTransactions();
+      }
```

Comment:
**Question:** This feels weird to me. Why would the user of the model need to know that the model has pending transactions, and then tell it to deal with them?